web/twa-vis-platform/code/src/ui/map/mapbox/mapbox-icon-loader.ts

```diff
@@ -29,11 +28,14 @@ export async function addIcons(map: Map, iconSettings: IconSettings): Promise<vo
  * @param callback callback function
  */
 function loadIcon(map: Map, imageName: string, imageURL: string, callback: () => void) {
+  if (process.env.ASSET_PREFIX) {
+    console.info(`ASSET_PREFIX: "${process.env.ASSET_PREFIX}" detected. Appending to MapBox Layer icon paths`)
```

Comment:
Do we wish to log this? This will log for each icon so it could be a lot of logs